pkgs/tools/misc/minipro/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, pkg-config
+, libusb1
+, which
+}:
+
+stdenv.mkDerivation rec {
+  pname = "minipro";
+  version = "0.5";
+
+  src = fetchFromGitLab {
+    owner = "DavidGriffith";
+    repo = "minipro";
+    rev = version;
+    sha256 = "sha256-Hyj2LyY7W8opjigH+QLHHbDyelC0LMgGgdN+u3nNoJc=";
+  };
+
+  nativeBuildInputs = [ pkg-config which ];
+  buildInputs = [ libusb1 ];
+  makeFlags = [ "DESTDIR=$(out)" "PREFIX=" "CC=cc" ];
```

Comment:
setting `PREFIX=` empty removes the install of udev rules, which are convenient to get the programmer accessible from a user.

Reply:
by coincidence I made another derivation at the same time and I found:
https://github.com/NixOS/nixpkgs/pull/121836/files#diff-474178633c840f49ad63ac5037ec42ac6053ce906ad33b93f7a922aa21e6b327R21

to work pretty nicely.

Reply:
Thanks for the feedback. I committed a slightly modified version to pick up the udev rules and bash completions.